1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,32 @@
 function readingList(books) {
   // Write your code here...
+
+  for (let book of books) {
+    const paragraph = document.createElement("p");
+    // let content = document.querySelector("#content");
+    // content.appendChild(paragraph);
+    paragraph.innerText = `${book.title} by ${book.author}`;
+
+    const li = document.createElement("li");
+    let ul = document.querySelector("#reading-list");
+
+    if (book.alreadyRead) {
+      li.style.backgroundColor = "red";
+    } else {
+      li.style.backgroundColor = "green";
+    }
+    li.appendChild(paragraph);
+    ul.appendChild(li);
+
+    let bookImage = document.createElement(`img`);
+    bookImage.src = book.bookCoverImage;
+    paragraph.appendChild(bookImage);
+  }
+
+  // document.querySelector("#reading-list").appendChild(li);
+  // li.appendChild(title);
+  // li.appendChild(author);
+  // li.appendChild(image);
 }
```

Comment:
Hi Saqib - really nice effort! just a couple of small points:

1. Please tidy up your codes by deleteing the codes you have added and then are commented out. It helps to reduce clutter and make it easier to maintain the code over time, also a help to avoid confusion!
2. As the instruction says, could you please use CSS classes created in the styles.css file to set background color? by using CSS classes here, if you need to change the style later, you can update the CSS class once and all elements will be updated automatically and would be a good practice to figure out how to use CSS stuff in DOM.

Reply:
ok i will do that